extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js

```diff
@@ -67,12 +67,16 @@ define('ext.wikia.adEngine.provider.factory.wikiaGpt', [
 			});
 		}
 
+		function getExtraAdUnitBuilder() {
+			return typeof extra.adUnitBuilder === 'function' ? extra.adUnitBuilder() : extra.adUnitBuilder;
```

Comment:
Wouldn't it be easier to change all `extra.adUnitBuilder` to  `extra.getAdUnitBuilder` and use it as function everywhere?